### ISSUE TITLE
shift-click not working on seeing_profile and tabs not displaying properly

### ISSUE THREAD

**Issue by JuanCab** [feder-observatory/stellarphot#93]
The current version of the code from GitHub (pre-astro 5.3 patch) running on my Mac is having two problems with `seeing_profile` 
1. right-click is NOT properly triggering the construction of the TESS Seeing Profile (a button appears, but the seeing profile is not present).  
2. The SNR/Seeing Profile/Integrated Counts tabs DO NOT appear, instead the text appears all mashed together (See image linked from first comment below).  Problem occurs whether run in JupyterLab or VSCode running Jupyter notebook extensions.

The version running on our Jupyter3 server doesn't appear to have an issue.  Emily and I were able to extract the stellarphot codebase used on Jupyter3, and comparing it to the current codebase there seemed to be NO relevant differences in `stellarphot.visualization.seeing_profile_functions.py` in the two versions (e.g. - only differences was in `__all__` list and in `np.int` being replaced with `int` in the newer code).

**Comment by JuanCab** [comment]
I was able to get a bit of information on the conda environment on jupyter3 and set up the following conda environment:
```
name: jupyter3
channels:
  - conda-forge
dependencies:
  - aggdraw=1.3.15
  - astropy=5.0.4
  - astroquery=0.4.6
  - astrowidgets=0.1.1
  - batman-package=2.4.9
  - bottleneck=1.3.5
  - ccdproc=2.3.1
  - gatspy=0.3
  - ginga=3.4.0
  - ipyfilechooser=0.6.0
  - ipython=8.4.0
  - ipywidgets=7.7.2
  - jupyterlab=3.4.5
  - matplotlib=3.5.3
  - numpy=1.23.2
  - pandas=1.4.4
  - photutils=1.5.0
  - python=3.9.7
  - pyyaml=6.0
  - scipy=1.9.1
prefix: ~/miniconda3/envs/jupyter3
```

Running the notebook with this environment SHOWED the tabs properly for the `seeing_profile.box` call (although the actual flux profile still didn't display).

**Comment by JuanCab** [comment]
I confirmed that this environment, which has everything up to date EXCEPT `jupyterlab` does display tabs properly:
```
name: SNdev
channels:
  - conda-forge
dependencies:
  - aggdraw
  - arrow
  - astropy
  - astroquery
  - astrowidgets
  - batman-package
  - bottleneck
  - ccdproc
  - fqdn
  - gatspy
  - ginga
  - graphviz
  - ipyfilechooser
  - ipython
  - ipywidgets
  - isoduration
  - jsonpointer
  - jupyter
  - jupyterlab=3.4.5
  - matplotlib
  - numpy
  - pandas
  - photutils
  - pytest
  - python=3.11.*
  - pyyaml
  - scipy
  - sphinx-astropy
  - tox
  - uri-template
  - webcolors
prefix: ~/miniconda3/envs/SNdev
```

This suggests that unfortunately the problem is probably with how one of the widget libraries (either `astrowidgets` or `ipywidgets` interacts with the latest `jupyterlab`).

**Comment by JuanCab** [comment]
I submitted an [issue](https://github.com/jupyter-widgets/ipywidgets/issues/3785) to JupyterWidgets about the tabs not having boundaries on JupyterLab>=4.0 a week ago. Looks like today Andy Maloney and Martin Renou at JupyterWidgets have figured out the issue with tabs and resolved it with [JupyterLab PR #3787](https://github.com/jupyter-widgets/ipywidgets/pull/3787).

**Comment by JuanCab** [comment]
Fixed in ipywidgets 8.0.7
